### PR TITLE
SES user profile view

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket
189-feature-useraccounts-v1

# Description

# Actions
Add view user-profile
Add missing Label manager profile

### CODE REVIEW

File: pages/auth/user-profile/index.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import { featureFlags } from '../../../feature-flags/feature-flags';
+import { CURRENT_ENVIRONMENT } from '../../../src/config/endpoints';
+import { useAuthContext } from '../../../src/core/context/AuthContext';
+import UserProfile from '../../../src/stories/containers/users/user-profile/user-profile';
+import NotFoundPage from '../../404';
+
+const UserProfilePage: NextPage = () => {
+  const { authToken } = useAuthContext();
+
+  if (!featureFlags[CURRENT_ENVIRONMENT].FEATURE_AUTH || !authToken) {
```
Comment (YasielCabrera) [inline pages/auth/user-profile/index.tsx:12]: this should be handled in SSR using `getServerSideProps` instead returning not found in order to let nextjs handle the 404 error

File: src/stories/containers/users/user-profile/user-profile.tsx [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+  const { isLight } = useThemeContext();
+  const { user, clientRequest, clearCredentials } = useAuthContext();
+  const [checked, setChecked] = useState(false);
+  const isMobileOrTable = useMediaQuery(lightTheme.breakpoints.between('table_375', 'desktop_1194'));
+
+  const handleChange = useCallback(async () => {
+    const { query: gqlQuery, input } = ENABLE_DISABLE_USER_REQUEST(!checked, '1');
+    const data = await clientRequest?.request(gqlQuery, input);
+    if (data) {
+      console.log('data', data.userSetActiveFlag[0].active);
```
Comment (YasielCabrera) [inline src/stories/containers/users/user-profile/user-profile.tsx:33]: is this `console.log` needed here?

File: src/stories/containers/users/user-profile/user-profile.tsx [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+            <UserLabelValue isLight={isLight}>{user?.username}</UserLabelValue>
+          </div>
+          <ContainerPassword>
+            <UserNameLabel isLight={isLight}>Password:</UserNameLabel>
+            <div
+              style={{
+                display: 'flex',
+                flexDirection: 'row',
+                alignItems: 'center',
+              }}
```
Comment (YasielCabrera) [inline src/stories/containers/users/user-profile/user-profile.tsx:77]: why no use styled-components in those divs instead of inline styles?

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: src/stories/containers/users/user-profile/user-profile.tsx (hunk 3 above)
Comment (alejandrocabriales) [reply src/stories/containers/users/user-profile/user-profile.tsx:77]: You are right , I will use it

File: src/stories/containers/users/user-profile/user-profile.tsx (hunk 2 above)
Comment (alejandrocabriales) [reply src/stories/containers/users/user-profile/user-profile.tsx:33]: Not my but, I will remove

File: pages/auth/user-profile/index.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply pages/auth/user-profile/index.tsx:12]: In fact it was my mistake , its not a page, so I will remove from the page directory .

**Comment by YasielCabrera** [comment]
LGTM

**Pull request merged**